tools/triage/clean_old_issues.py

```diff
@@ -60,6 +62,7 @@ def fetch_issues_from_jira(
     return list(result_list)
 
 
+@retry(exceptions=jira.JIRAError, tries=3, delay=10)
```

Comment:
it's retrying the whole process again from the start (for all issues), so I'm not sure it will get the affect we want

Reply:
you also might try to re-close issues, I think it might raise some errors